ts/src/base/Exchange.ts

```diff
@@ -476,11 +476,15 @@ export default class Exchange {
                 'createMarketOrderWithCost': undefined,
                 'createMarketSellOrderWithCost': undefined,
                 'createOrders': undefined,
+                'createOrderWithTpAndSl': undefined,
```

Comment:
I'm wondering if we should make the name more explicit, like 
createOrderWithStopLossAndTakeProfit

Reply:
`createOrderWithStopLossAndTakeProfit` feels a bit verbose but I think it's a good idea to make them more explicit